openedx/openedx-platform
# fix: tooltip positioning near edge [FC-0062]

## Description

This PR fixes the tooltip positioning rule to avoid rendering it outside the window/iframe.

### Before
![image](https://github.com/user-attachments/assets/1e8c31e4-5ee6-470e-a9cb-35a6c22b5eda)
![image](https://github.com/user-attachments/assets/1dd9e859-942f-4eb1-9060-370dba69fe85)

### After
![image](https://github.com/user-attachments/assets/53c663fd-3fce-46c5-afa7-0e1ddb219e66)
![image](https://github.com/user-attachments/assets/3a49ad61-9974-405d-87a5-4c0777a78979)

## Additional Infomation
- Part of https://github.com/openedx/frontend-app-authoring/issues/1463

## Testing Instructions

1. Open the Unit Outline in the course authoring MFE
2. Open the browser console and edit the DOM adding a `data-tooltip` attribute to any element near the left edge of the screen like the example below: 
![image](https://github.com/user-attachments/assets/e6800f10-7f6c-40e2-89cd-0df5d1396711)
3. Put the mouse over the element an check if it is rendered inside the screen/iframe limit
4. Repeat the process with an element near the right edge
___
Private ref: [FAL-3941](https://tasks.opencraft.com/browse/FAL-3941)
